pytorch_lightning/utilities/model_helpers.py

```diff
@@ -69,3 +71,28 @@ def is_overridden(
     parent_code = parent_attr.__code__
 
     return instance_code != parent_code
+
+
+def get_model_size(model: nn.Module) -> int:
```

Comment:
1. this could be under https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/utilities/memory.py
2. `get_model_size_bytes` is explicit that the return will be in bytes

Reply:
Should utility be part of the training_type_plugin as shared tensors won't provide the right value ?